src/data/faq.json

```diff
@@ -199,7 +199,12 @@
+                            "Your phone settings display the exposure checks for the last 14 days:",
+                            "<ul><li><b>iOS:</b> 'Settings' -> 'Exposure Notifications' -> 'Exposure Logging Status' -> 'Exposure Checks'</li><li><b>Android:</b> 'Settings' -> 'Google' -> 'COVID-19 exposure notifications' -> 'Exposure Checks'</li></ul>",
+                            "When the app was running constantly during the last two weeks, you should see at least one check every day for all of the past 14 days.",
+                            "Please note: The log is just the history. After every background update, the exposures of the last 14 days are checked and the risk status calculated. The important thing is that the exposure log shows at least one check for the last 24 hours.",
+                            "It might be that entries are displayed for less than 14 days, although exposure was checked correctly. There are several reasons:",
+                            "<ol><li>The log has a fixed limit of 100 entries on iOS devices. If there are more entries, only the latest are kept. We're in contact with Apple so that the complete log will be displayed. The log limit of 100 entries on Android devices has been recently removed by Google at the end of August 2020.</li><li>If you use the app only recently (let's say for 5 days), only checks that were made after setting up the app are displayed - at least one for every day that the Corona-Warn-App was running constantly.</li></ol>"
```

Comment:
Suggest changing:

The log limit of 100 entries on Android devices has been recently removed by Google at the end of August 2020.

to:

Android devices are limited to 300 entries.